bundle-generator/deployment/src/main/java/io/quarkiverse/operatorsdk/bundle/deployment/BundleProcessor.java

```diff
@@ -127,6 +139,37 @@ CSVMetadataBuildItem gatherCSVMetadata(KubernetesConfig kubernetesConfig,
         return new CSVMetadataBuildItem(csvGroups);
     }
 
+    private static String getDefaultProviderURLFromSCMInfo(ApplicationInfoBuildItem appConfiguration,
+            JarBuildItem jarBuildItem) {
+        final var maybeProject = KubernetesCommonHelper.createProject(appConfiguration, Optional.empty(),
```

Comment:
@metacosm need to be extremely careful here. This class uses Dekorate to get the URL info directly from local git config. Personally I consider this unexpected (compared to using VCS info from pom configuration) and it was also the source of the recent Quarkus CVE .